03week/mastermind.js

```diff
@@ -20,43 +83,83 @@ function printBoard() {
 function generateSolution() {
   for (let i = 0; i < 4; i++) {
     const randomIndex = getRandomInt(0, letters.length);
-    solution += letters[randomIndex];
+    generatedCode += letters[randomIndex];
   }
 }
 
 function getRandomInt(min, max) {
   return Math.floor(Math.random() * (max - min)) + min;
 }
 
-function generateHint() {
-  // your code here
+function mastermind(guess) {
+  if (guess === generatedCode) {
+    console.log('You win!');
+    return;
```

Comment:
You need to return  your "win" statement here, instead of just console logging here.

See line 161 to test instructions.